spec/features/realtime_data/sip_options_probers/index_sip_options_probers_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Sip Options Probers index page', js: true do
+  include_context :login_as_admin
+
+  subject do
+    visit sip_options_probers_path
+    stub_jrpc_request('options_prober.show.probers', nodes.first.rpc_endpoint, { logger: be_present }).and_return([{ id: nodes.first.id }])
+    stub_jrpc_request('options_prober.show.probers', nodes.second.rpc_endpoint, { logger: be_present }).and_return([{ id: nodes.second.id }])
+  end
+
+  let!(:nodes) { create_list(:node, 2) }
+  let(:jrpc_response) do
+    NodeRpcClient.perform_parallel(default: []) do |client, node|
+      result = client.sip_options_probers
+      result.map { |row| row.merge(node: node) }
+    end
```

Comment:
you should not use NodeRpcClient in this test

you should have let with stubbed response from nodes RPC which you should use in stub_jrpc_request and expectations